### DISCUSSION TITLE
is `RouterContext` disappeared?

### DISCUSSION THREAD

**Discussion by mattiaz9** [TanStack/router#795] [Q&A] [answered]
i updated to `0.0.1-beta.205` and now I get the error: `Module '"@tanstack/react-query"' has no exported member 'RouterContext'`.

Is this a bug or it has just been removed?

**Comment by tannerlinsley** [comment, marked as answer]
Renamed to RouterMeta so it don’t get confused with the React context that powers the router.

Tanner Linsley

**Reply by mattiaz9** [reply]
ok thanks 👍
